base/Ballerina.toml

```diff
@@ -22,3 +17,4 @@ version = "1.0.3"
 org = "ballerinax"
 name = "health.fhir.r4utils.fhirpath"
```

Comment:
Do we need this dependency now?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/77219fd0e700d692f1ec59715acd5ff1ea1e3848